src/routes/globe/(components)/Globe.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import createGlobe from 'cobe';
+
+    let canvas: HTMLCanvasElement;
+    let phi = 0;
+
+    onMount(() => {
+        let width = 0;
+        const onResize = () => canvas && (width = canvas.offsetWidth);
+        window.addEventListener('resize', onResize);
+        onResize();
+        const globe = createGlobe(canvas, {
```

Comment:
I think a svelte action might be better/faster here, it will be immediatly executed once the node is ready. you also dont need the bind:this for the element anymore 👍🏻 

https://svelte.dev/docs/svelte-action